elixir_server/lib/elixir_server/ocr.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Ocr do
+  def pass(_outputOcr) do
+    true
+  end
+
+  def hasText(outputOcr) do
+    String.length(outputOcr) > 0
+  end
+
+  def containsText(text, outputOcr) do
+    String.contains?(outputOcr, text)
+  end
+
+  def compareLength(operator, number, outputOcr) do
+    case is_integer(number) do
+      true ->
+        operator.(String.length(outputOcr), number)
+
+      false ->
+        IO.puts("Is not a number #{number}")
+        &pass/1
+    end
+  end
+
+  def minLength(number, outputOcr) do
+    compareLength(&Kernel.>=/2, number, outputOcr)
+  end
+
+  def maxLength(number, outputOcr) do
+    compareLength(&Kernel.<=/2, number, outputOcr)
+  end
+
+  def parseOption(option) do
+    {name, param} = option
+
+    case name do
+      "hasText" ->
```

Comment:
do we want this param? Shouldn't it be that the default behavior (without params) of this program is to send back image paths with text?

Reply:
added parsing true/false